standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/messaging/MessageFactory.java

```diff
@@ -107,8 +106,7 @@ private static MessageFactory getInstance(String className) {
    */
   public static MessageDeserializer getDeserializer(String format,
                             String version) {
-    return getInstance(hiveConf.get(CONF_LABEL_HCAT_MESSAGE_FACTORY_IMPL_PREFIX + format,
```

Comment:
Previously it was instantiating CONF_LABEL_HCAT_MESSAGE_FACTORY_IMPL_PREFIX which is different than the EVENT_MESSAGE_FACTORY

Reply:
Ok, will fix.

Reply:
Now I see why I did this.  Based on these comments near the beginning of the file:
// This parameter is retained for legacy reasons, in case someone implemented custom
  // factories. This, however, should not be the case, since this api was intended to
  // be internal-only, and we should manage the jms and json implementations without
  // needing this parameter. Marking as deprecated, for removal by 2.4 - see corresponding
  // note on the getDeserializer(String,String) method
Since it was marked for removal in 2.4 I removed it since we're on master (3.0).